src/SeqCli/PlainText/LogEvents/LogEventBuilder.cs

```diff
@@ -35,7 +35,7 @@ public static LogEvent FromProperties(IDictionary<string, object?> properties, s
         var messageTemplate = GetMessageTemplate(properties);
         var traceId = GetTraceId(properties);
         var spanId = GetSpanId(properties);
-        var props = GetLogEventProperties(properties, remainder, level);
+        var props = GetLogEventProperties(properties, remainder);
```

Comment:
This `level` is no longer needed; it would tack the surrogate level property on, but we do this while reading now.